conda/recipes/custreamz/meta.yaml

```
@@ -19,15 +19,17 @@ build:
   script_env:
     - VERSION_SUFFIX
     - PARALLEL_LEVEL
-    - CC
-    - CXX
-    - CUDAHOSTCXX
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 {{ sysroot_version }}  # [linux64]
+    - sysroot_linux-aarch64 {{ sysroot_version }}  # [aarch64]
   host:
+    - cudf_kafka {{ version }}
```

Does custreamz not need cudf? It's just a transitive dependency via cudf_kafka? I realize this isn't a change, but since the cudf/meta.yaml does look to have been changed meaningful I want to double-check this one as well.

---

Yes, `cudf_kafka` depends on `cudf`, so `custreamz` depends on `cudf` too

---

On that note, why does `cudf` match to `{{ version }}.*` but `cudf_kafka` only matches `{{ version }}`?
```suggestion
    - cudf {{ version }}.*
    - cudf_kafka {{ version }}
```

---

I tried to unify the recipes without changing everything, but all the recipes should match `{{ version }}.*`, but I think we should do this in another PR. That's on my TODO list